themes.json

```diff
@@ -350,5 +350,37 @@
         "/posts/instructions.md"
       ]
     }
+  },
+  {
+    "id": "pro-file",
+    "name": "pro-file",
+    "description": "A theme to create a simple resume website.",
+    "tags": ["cv", "resume"],
+    "lume_version": 2,
+    "author": {
+      "name": "Nap Joseph Calub",
+      "url": "https://napjose.ph"
+    },
+    "repo": "https://github.com/wewillcraft/pro-file",
+    "demo": "https://pro-file.willcraft.software",
+    "screens": [
+      {
+        "desktop": "/screens/pro-file/desktop.png",
+        "mobile": "/screens/pro-file/mobile.png"
+      }
+    ],
+    "module": {
+      "name": "pro-file",
+      "id": "denoland/pro_file",
+      "origin": "https://deno.land/x/pro_file",
+      "main": "/mod.ts",
+      "cms": "/_cms.ts",
+      "srcdir": "/test",
```

Comment:
Your `src` directory is `/src`, it's where your source files are stored.

Reply:
Hi, @oscarotero! Should I change it to `"/src"`?

My original understanding was that given the said config, the init script will copy

- https://github.com/wewillcraft/pro-file/blob/main/test/favicon.svg
- https://github.com/wewillcraft/pro-file/blob/main/test/_data.yml
- https://github.com/wewillcraft/pro-file/blob/main/test/index.yml

to the user's newly created project directory. Was it not the case?

Reply:
Ah, ok. I didn't realize you moved the files to that folder.

In my themes I have everything in the `src` folder (remote files and copied files. You can see the [simple blog](https://github.com/lumeland/theme-simple-blog/tree/main/src), for instance).
This allows to run the theme as a normal site from the root folder. For example, pro-file now fails if I run `deno task serve` from the root because these files are missing in the src folder.

The `test` folder was only for testing purposes, to make sure the theme works when it's loaded remotely. And can be used also for testing with different content without affecting the original files.

Reply:
Okay, I moved the files to `src` now https://github.com/wewillcraft/pro-file/commit/6aa48af0cc65cb2ac2b473dbbd83f9a96dda36c9.

My `test/_config.ts` will now have:

```
site.use(theme());
const files = [
  "_data.yml",
  "favicon.svg",
  "index.yml",
];
for (const file of files) {
  site.remoteFile(file, import.meta.resolve(`../src/${file}`));
}
```

So that I don't have to duplicate those files. Does that work?

Reply:
Yes, it should work.

Maybe the `test` folder wasn't a good idea because it's confusing. It was just a way to test the theme, not intended to publish the content.
I have to think in a better way to test themes.